packages/yumemi_lints/pubspec.yaml

```diff
@@ -6,6 +6,15 @@ repository: https://github.com/yumemi-inc/flutter-yumemi-lints
 environment:
   sdk: '>=2.17.0 <4.0.0'
 
+dependencies:
+  args: ^2.3.1
+  meta: ^1.10.0
+  path: ^1.8.3
+  pub_semver: ^2.1.4
+
+dev_dependencies:
+  test: ^1.21.0
```

Comment:
![imo-badge](https://img.shields.io/badge/review-imo-skyblue.svg) 
It looks good to use test 1.21.4.
```suggestion
  test: ^1.21.4
```